### DISCUSSION TITLE
pg_receivewal: error: unexpected termination of replication stream: ERROR:  requested starting point 0/7000000 is ahead of the WAL flush position of this server 0/16F9DE0

### DISCUSSION THREAD

**Discussion by skpupil** [pgmoneta/pgmoneta#67] [Q&A]
Has anyone got this error?
```
pg_receivewal: error: unexpected termination of replication stream: ERROR:  requested starting point 0/7000000 is ahead of the WAL flush position of this server 0/16F9DE0
pg_receivewal: error: disconnected
```
I go through the pgmoneta [tutorial](https://github.com/skpupil/pgmoneta-plus/tree/pgmoneta-plus-dev/doc/tutorial) carefully. But this error is seems from postgres.

**Comment by skpupil** [comment]
I solved. This is because the backup file in base_dir is partial. I erased it.

**Reply by jesperpedersen** [reply]
Meaning ?

Seems like PostgreSQL and pgmoneta came out of sync - can you reproduce it ?

**Reply by skpupil** [reply]
I followed your tutorial and did it step by step. After that, I init another new database. During the replication process, pgmoneta shows that the replication data location of pgmoneta is not match  that of postgres. Then, I delete the replication data if pgmoneta, and started it again. Finally, it worked.

**Reply by jesperpedersen** [reply]
Ok.

Yeah, pgmoneta requires that its kept in sync with the cluster.
